### ISSUE TITLE
[BUG] Breaks latex parsing when auto on enabled

### ISSUE THREAD

**Issue by Bpklgs** [ansh/jiffyreader.com#259]
Kind of hard to describe but bolding pages before they load breaks parts of them

**Reproduce on the extension**
1. Enable auto turn on
2. Go to https://courseware.cemc.uwaterloo.ca/42/assignments/1139/1
3. Scroll down a bit and you should see text that shouldn't be there
![image](https://github.com/ansh/jiffyreader.com/assets/20851930/b2e53134-1b0a-4013-865d-f830911889a0)

**Expected behavior**
go to the website and turn on the extension manually

**Desktop (please complete the following information):**
 - OS: Windows10
 - Browser: Opera gx
 - Extension version: 1.0.10.1

**Comment by asieduernest12** [comment]
@Bpklgs please include a screen shot if you can to help with the investigation into the issue

**Comment by asieduernest12** [comment]
@asieduernest12 currently you cannot use auto turn-on with this page

the issue:
from the initial review, the page https://courseware.cemc.uwaterloo.ca/42/assignments/1139/1 is running some script to parse latex code and display them as math formula notations in human readable format

the auto turn on code runs immediately after the a page is ready
this is causing the latex code to be jiffied and no longer be valid latex
this means the latex parser is not seeing any valid latex code to transform

a intermetiate fix for this issue is to disable auto turn on for this site and then turn on jiffy manually after the page and latex is ready

an ideal programatic solution is to realease an update that will let you specify a trigger such checking for the presence or absence of some element which then allows auto turn on to engage

this maybe coupled with exponential backoff and retry limit to account for slow or degraded network conditions in order to provide a user experience that is acceptable with out introducing adverse penalties on user interactions

TLDR: disable auto turn on for https://courseware.cemc.uwaterloo.ca/42/assignments/1139/1 by using checking the customize for only this site and then disable auto turn on

**Comment by asieduernest12** [comment]
features to consider

- [ ] add support for default auto on triggers as js functions  placed in auto - on file
- [ ] make turn on code ternally configurable to accept user defined element exceptions scoped to domains
